client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -136,139 +121,163 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
-        </tbody>
-      </SettingsTable>
-      <ContestsTable>
-        <thead>
-          <tr>
-            <th>Target Contests</th>
-            <th>Jurisdictions</th>
-          </tr>
-        </thead>
-        <tbody>
-          {targetedContests.map((c: IContest) => (
-            <tr key={c.id}>
-              <td>{c.name}</td>
-              <td>{c.jurisdictionIds.join(', ')}</td>
+      <ElevatedCard>
+        <SettingsTable>
+          <tbody>
+            <tr>
+              <td>Election Name:</td>
+              <td>{electionName}</td>
             </tr>
-          ))}
-        </tbody>
-      </ContestsTable>
-      <ContestsTable>
-        <thead>
-          <tr>
-            <th>Opportunistic Contests</th>
-            <th>Jurisdictions</th>
-          </tr>
-        </thead>
-        <tbody>
-          {opportunisticContests.map((c: IContest) => (
-            <tr key={c.id}>
-              <td>{c.name}</td>
-              <td>{c.jurisdictionIds.join(', ')}</td>
+            <tr>
+              <td>Risk Limit:</td>
+              <td>{riskLimit}</td>
             </tr>
-          ))}
-        </tbody>
-      </ContestsTable>
-      <H4>Sample Size Options</H4>
+            <tr>
+              <td>Random Seed:</td>
+              <td>{randomSeed}</td>
+            </tr>
+            <tr>
+              <td>Participating Jurisdictions:</td>
+              <td>
+                <a
+                  href={`/api/election/${electionId}/jurisdiction/file/csv`}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {jurisdictionFile && jurisdictionFile.file
+                    ? jurisdictionFile.file.name
+                    : ''}
+                </a>
+              </td>
+            </tr>
+            <tr>
+              <td>Audit Board Data Entry:</td>
+              <td>{online ? 'Online' : 'Offline'}</td>
+            </tr>
+          </tbody>
+        </SettingsTable>
+        <ContestsTable>
+          <thead>
+            <tr>
+              <th>Target Contests</th>
+              <th>Jurisdictions</th>
+            </tr>
+          </thead>
+          <tbody>
+            {targetedContests.map((c: IContest) => (
+              <tr key={c.id}>
+                <td>{c.name}</td>
+                <td>{c.jurisdictionIds.join(', ')}</td>
+              </tr>
+            ))}
+          </tbody>
+        </ContestsTable>
+        <ContestsTable>
+          <thead>
+            <tr>
+              <th>Opportunistic Contests</th>
+              <th>Jurisdictions</th>
+            </tr>
+          </thead>
+          <tbody>
+            {opportunisticContests.map((c: IContest) => (
+              <tr key={c.id}>
+                <td>{c.name}</td>
+                <td>{c.jurisdictionIds.join(', ')}</td>
+              </tr>
+            ))}
+          </tbody>
+        </ContestsTable>
+      </ElevatedCard>
+      <br />
+      <H4>Estimated Sample Sizes</H4>
```

Comment:
I feel like this header is a bit misleading. The sample sizes are the actual sizes that are going to be used, they aren't just estimated. I feel like it should say "Select Sample Size" or just "Sample Size" or something like that. @mcchilders